src/libstd/sys/unix/process/process_common.rs

```diff
@@ -1,19 +1,27 @@
 use crate::os::unix::prelude::*;
 
-use crate::ffi::{OsString, OsStr, CString, CStr};
+use crate::ffi::{OsString, OsStr, CString};
 use crate::fmt;
 use crate::io;
 use crate::ptr;
 use crate::sys::fd::FileDesc;
-use crate::sys::fs::{File, OpenOptions};
+use crate::sys::fs::File;
 use crate::sys::pipe::{self, AnonPipe};
 use crate::sys_common::process::{CommandEnv, DefaultEnvKey};
 use crate::collections::BTreeMap;
 
+#[cfg(not(target_os = "fuchsia"))]
```

Comment:
Since this file is shared amongst most platforms, would it be possible to avoid the fuchsia-specific cfgs? Could `Null` be used on all platforms and only at the very end it's lowered to `/dev/null`?